source/_patterns/04-pages/03-teaser.mustache

```diff
@@ -21,29 +21,84 @@
     {{/ teaserContent50List }}
   </div>
 
-
   <h2>Teaser offer</h2>
 
-  <h3>Teaser offer 25</h3>
-
+  <h3>Teaser offer 50</h3>
   <div class="teaser-list">
-    {{# teaserOffer25List }}
-    {{/ teaserOffer25List }}
+    <div class="teaser teaser-50 teaser-offer">
+      <a href="#" class="teaser-box" role="button">
+        {{> molecules-teaser-images }}
+        <div class="teaser-body">
+          {{> molecules-teaser-headline }}
+          <p class="teaser-description">
+            "Dies ist ein <strong>richtig cooler Hotelname</strong>&nbsp;{{> molecules-hotel-category-images}}, wie Restmüllbehältervolumenminderungsgesetz Restmüllbehältervolumenminderungsgesetz Restmüllbehältervolumenminderungsgesetz Restmüllbehältervolumenminderungsgesetz
+          </p>
+          <div class="teaser-cta">
+            {{> molecules-button-price}}
+          </div>
+        </div>
+      </a>
+    </div>
 
-    {{^ teaserOffer25List }}
-      <p>... soming soon ...</p>
-    {{/ teaserOffer25List }}
-  </div>
+    <div class="teaser teaser-50 teaser-offer">
```

Comment:
what's the difference between this teaser and...

Reply:
This is a more a strategic implementation of CSS. 
See it has teaser implements styling of teaser-50 and teaser-offer
I think, this is the best way, because we do the teasers depending on what they are and what size they have.

A general teaser has following properties. Every teaser has it. No matter the what and the size:

```.teaser {
  margin-bottom: $grid-gutter-width;
  padding-left: $grid-gutter-width / 2;
  padding-right: $grid-gutter-width / 2;
  width: 100%;
}
```

A teaser-50 describes the following properties. Defining the size. This is for breakpoint MD.

```.teaser-50 {
    flex: 0 0 50%;
    max-width: 50%;
  }
```

I can not tell what the class teaser-offer means. Since it wasn't me who implemented it, I can not tell if the class is being used in a different microsystem, for example TOM.